articles/anomaly-detection/index.md

```diff
@@ -44,8 +42,8 @@ Auth0 does email the dashboard owner when this block is triggered. Within this e
 
 Both of these anomaly types depend on the IP address of the user. Because of this, the following use cases are not supported:
 
-1.  Using the [Resource Owner](/auth-api#!#post--oauth-ro) from the backend of the application. Using this call does not get the IP address of the user.
-2. Authenticating many users from the same IP address.  If this is an issue for you, disable the **2nd level brute force protection** shield.
+1. Using the [Resource Owner](/auth-api#!#post--oauth-ro) from the backend of the application. Using this call does not get the IP address of the user.
+2. Authenticating many users from the same IP address. For users that are behind a proxy is very likely to reach this limit, and trigger this protection. It's possible to configure a whitelist for this IPs and CIDR ranges. 
```

Comment:
Authenticating many users from the same IP address. Users that are behind a proxy are likely to reach this limit, and trigger this protection. It's recommended to configure a whitelist for the proxy's IP and CIDR range.